sympy/stats/tests/test_rv.py

```diff
@@ -439,3 +439,12 @@ def test_issue_20286():
     k = Dummy('k', integer = True)
     eq = Sum(Piecewise((-p**k*(1 - p)**(-k + n)*log(p**k*(1 - p)**(-k + n)*binomial(n, k))*binomial(n, k), (k >= 0) & (k <= n)), (nan, True)), (k, 0, n))
     assert eq.dummy_eq(H(B))
+
+def test_issue_20669():
+    from sympy import N
```

Comment:
Imports go at the top of the file but actually we don't need `N` here because we can just use `evalf` instead.